models/event.js

```diff
@@ -1,12 +1,21 @@
-'use strict';
+"use strict";
+const User = require("./user");
 module.exports = (sequelize, DataTypes) => {
-  const Event = sequelize.define('event', {
-    name: DataTypes.STRING,
-    pin: DataTypes.STRING,
-    description: DataTypes.TEXT
-  }, {});
-  Event.associate = function(models) {
-    // associations can be defined here
+  const Event = sequelize.define(
+    "event",
+    {
+      name: DataTypes.STRING,
+      date: DataTypes.STRING,
```

Comment:
Just wondering, if this means the date of an event, can there be event that spans multiple days?